js/picc.js

```diff
@@ -1,5 +1,5 @@
+!function(exports){function getTrigger(root){var selector=root.getAttribute("trigger")||"[aria-controls]",trigger=root.querySelector(selector);return trigger&&trigger.hasAttribute("aria-controls")?trigger:void console.warn('no trigger found for selector: "'+selector+'"')}function getContent(root,trigger){var contentId=trigger.getAttribute("aria-controls"),content=document.getElementById(contentId);return content?content:void console.warn('no content element found with id: "'+contentId+'"')}function getAriaBoolean(el,attr){return el&&el.hasAttribute(attr)&&"false"!==el.getAttribute(attr)}var EXPANDED="aria-expanded",HIDDEN="aria-hidden",TOGGLE_EVENTS=["click"];exports.ARIAAccordion=document.registerElement("aria-accordion",{prototype:Object.create(HTMLElement.prototype,{attachedCallback:{value:function(){var trigger=this.__trigger=getTrigger(this);if(trigger){var content=this.__content=getContent(this,trigger);if(content){trigger.setAttribute("tabindex",0),"BUTTON"===trigger.nodeName||trigger.hasAttribute("role")||trigger.setAttribute("role","button"),trigger.hasAttribute(EXPANDED)||trigger.setAttribute(EXPANDED,getAriaBoolean(this,EXPANDED));var toggle=this.toggle.bind(this),toggleAndCancel=this.__toggleAndCancel=function(e){return toggle(),e.preventDefault(),!1};TOGGLE_EVENTS.forEach(function(event){trigger.addEventListener(event,toggleAndCancel)}),this.update()}}}},detachedCallback:{value:function(){var trigger=this.__trigger;if(trigger){var toggleAndCancel=this.__toggleAndCancel;TOGGLE_EVENTS.forEach(function(event){trigger.removeEventListener(event,toggleAndCancel)})}}},update:{value:function(){var expanded=this.expanded;this.setAttribute(EXPANDED,expanded),this.__trigger.setAttribute(EXPANDED,expanded),this.__content.setAttribute(HIDDEN,!expanded)}},expanded:{get:function(){return getAriaBoolean(this.__trigger,EXPANDED)},set:function(value){value=!!value,value!==this.expanded&&this.__trigger&&(this.__trigger.setAttribute(EXPANDED,value),this.update(),this.dispatchEvent(new CustomEvent(value?"open":"close")))}},open:{value:function(){this.expanded=!0}},close:{value:function(){this.expanded=!1}},toggle:{value:function(){this.expanded=!this.expanded}},trigger:{get:function(){return this.__trigger}},content:{get:function(){return this.__content}}})})}(window);
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Missing semicolon.<br>Expected an assignment or function call and instead saw an expression.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.